package/src/plugins/add-integration.ts

```diff
@@ -0,0 +1,23 @@
+import { definePlugin } from "../core/define-plugin.js";
+import {
+	addIntegration,
+	type addIntegrationParams,
+} from "../utilities/add-integration.js";
+
+export const addIntegrationPlugin = definePlugin({
+	name: "addIntegration",
+	hook: "astro:config:setup",
+	implementation:
+		({ updateConfig, config, logger }) =>
+		(
```

Comment:
wdyt about calling it like so?
```ts
addIntegration(Vue(), {
	ensureUnique: false
})
```

Reply:
With the second argument being an object for the utility or the plugin, or both?? That's what I initially had, so it kept the `addIntegration(Vue())` API which I thought worked really nicely but then yeah pass an object